docs/tutorial_basics.rst

```diff
@@ -194,6 +194,39 @@ type of data structure is flagged at the instrument level with the attribute
 ``inst.pandas_format``, True if a DataFrame is returned by the corresponding
 instrument module load method.
 
+pysat also supports loading data from a range of files/file dates. Keywords
+in pysat with `end_*` are an exclusive bound, similar to slicing numpy arrays,
+while those with `stop_*` are an inclusive bound.
+
+.. code:: python
+
+   # load by year, day of year from 2001, 1 up to but not including 2001, 3
+   dmsp.load(2001, 1, end_yr=2001, end_doy=3)
```

Comment:
This command fails for me:

```
~/Programs/Git/pysatMadrigal/pysatMadrigal/instruments/methods/madrigal.py in load(fnames, tag, inst_id, xarray_coords, file_type)
    280                 data = xr.merge(fdata)
    281             else:
--> 282                 data = pds.merge(fdata)
    283 
    284     return data, meta

TypeError: merge() missing 1 required positional argument: 'right'
```

Reply:
This will likely be an issue in other places where `pds.merge` is used.

Reply:
Switching to
```
data = pds.concat(fdata)
```
works

```
In [6]: dmsp = pysat.Instrument(platform='dmsp', name='ivm', tag='utd', inst_id='f12', user=username, password=password)                                                                                                    

In [7]: dmsp.load(2001, 1, 2001, 3)                                                                                                                                                                                         

In [8]: dmsp.index                                                                                                                                                                                                          
Out[8]: 
DatetimeIndex(['2001-01-01 03:15:05', '2001-01-01 03:15:09',
               '2001-01-01 03:15:13', '2001-01-01 03:15:17',
               '2001-01-01 03:15:21', '2001-01-01 03:15:37',
               '2001-01-01 03:15:41', '2001-01-01 03:16:14',
               '2001-01-01 03:16:42', '2001-01-01 03:17:14',
               ...
               '2001-01-02 23:42:53', '2001-01-02 23:43:01',
               '2001-01-02 23:43:05', '2001-01-02 23:43:13',
               '2001-01-02 23:43:33', '2001-01-02 23:45:17',
               '2001-01-02 23:45:33', '2001-01-02 23:46:09',
               '2001-01-02 23:46:13', '2001-01-02 23:59:29'],
              dtype='datetime64[ns]', length=2620, freq=None)
```

Reply:
Check out pysat/pysatMadrigal#26 for the change.